README.md

```diff
@@ -25,9 +25,28 @@ then you just need to set deck default padding
 private val deckPager by lazy { activity.findViewById<Deck>(R.id.deck_pager) }
 
 private fun initDeck() {
-    deckPager.useDefaultPadding(activity)
+    deckPager.setDefaultPadding(activity)
 }
+```
+
+or set padding based on percentage of the screen width
+
+```
+// in kotlin
+deckPager.setPercentagePadding(activity, 8)
+
+// in XML
+app:padding_percentage="8"
```

Comment:
I think it's better to put xml properties in other code text block like

# Deck properties
```xml
<com.example.lib.Deck
        android:id="@+id/deck_pager"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:padding_percentage="8"
        app:padding_dp="16dp"/>
```

what do you think?

Reply:
Sure if you think it will improve clarity

Reply:
I believe you could edit it right away too :)